### PR TITLE
Multiple images in figure

### LINKED ISSUE #479: remark-captions: remove one image when Figure follows two images
# input

```
![Configuration du projet de test avec IntelliJ, on donne le nom "tuto" au projet puis on sélectionne maven en tant que système de build](/media/galleries/502/3b362168-4ea0-4698-b92b-319c902b8fe1.png)
![Résultat de la configuration : un arbre est créé avec le dossier .idea, le dossier src et le fichier pom.xml. Dans le dossier src on a un dossier main et un dossier test, ces deux derniers ont un dossier java](/media/galleries/502/b09e3d72-f0e6-4ba4-b4b4-d9a11a8a9981.png)
Figure: Configuration de notre projet et hiérarchie qui en résulte
```

# optained

Only one image is displayed 
image

# Expected 

Two possibilities : 

- the two images are displayed in the same figure mark
- two images = two figure marks

### PR DESCRIPTION
**Pull request by artragis**
This PR intents to enable multiple images in figures

fix #479 

For now it only fails with rebber as plugin relies on the assumption there is only one image

### CODE REVIEW

**Comment by StaloneLab** [comment]
From what I see, this PR is not ready to be merged. Am I right? If so, I will release a new version with your bugfixes for the escaping code, and this PR will be merged when ready. I will anyway make a review of the PR soon, to see what it does in more details.

**Comment by artragis** [comment]
This PR is not ready mainly due to the latex part. Il will ask some question to latex people before finishing it.

It also seems I, as usual, face a linter hell.

File: packages/remark-captions/__tests__/__snapshots__/index.js.snap [hunk 1]
```diff
@@ -7,6 +7,8 @@ exports[`#101 1`] = `
 <p><a href=\\"#\\"></a></p>"
 `;
 
+exports[`Two images in figure 1`] = `"<figure><img src=\\"img1.png\\" alt=\\"alt text1\\"><img src=\\"img2.png\\" alt=\\"alt text2\\"><figcaption>Common text</figcaption></figure>"`;
```
Comment (StaloneLab) [inline packages/remark-captions/__tests__/__snapshots__/index.js.snap:10]: I am not sure about a small detail: shouldn't every `img` have it's own `figure` tag? I was thinking about something like this, but couldn't find what the best practice was, especially in terms of accessibility:

```html
<figure>
  <figure>
    <img src="img1.png" />
  </figure>
  <figure>
    <img src="img2.png" />
  </figure>
  <figcaption>Global caption</figcaption>
</figure>

**Review by StaloneLab — COMMENTED** [review]
Fine by me for the HTML part, I haven't found the linter problems you were talking about. There were some tests failing that shouldn't have been, but running the tests again fixed the problem, the only one still failing is the LaTeX part of the test you added.

File: packages/remark-captions/__tests__/__snapshots__/index.js.snap (hunk 1 above)
Comment (artragis) [reply packages/remark-captions/__tests__/__snapshots__/index.js.snap:10]: in fact, figure should not have sub "figure" tag, the full figure is composed of two images and the global caption. So `<figure><figcaption></figure>` is ok.

As of latex, I will open a question on the template side to see if rendering the "standard way" is ok or if they need a specific command.

**Comment by StaloneLab** [comment]
What is the current state of this PR? Should I close it? It has been almost a year without activity.

**Comment by StaloneLab** [comment]
No news and no answer, closing for now, feel free to reopen if needed.

**Pull request closed without merging**